### PR TITLE
Fill triangle

### PR DESCRIPTION
**Pull request by sharvani28**
The following code demonstrates how to use the Fill_Triangle function to draw a simple red colour -filled triangle. It creates a triangle with specified coordinates and fills it with red color.

Here is the screenshot of how the Graphics Usage examples page look like.
grpahics page

### CODE REVIEW

**Review by s223126445 — APPROVED** [review]
The PR is about Fill Triangle.
Used top-level statements for both the codes.
I have tested the C# and C++ codes they are working fine.
C#
C++

File: src/assets/usage-examples-code/graphics/fill_triangle/fill_triangle.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### Fill Triangle
+
+The following code demonstrates how to use the Fill_Triangle function to draw a simple red colour -filled triangle. It creates a triangle with specified coordinates and fills it with red color.
```
Comment (hugglesfox) [inline src/assets/usage-examples-code/graphics/fill_triangle/fill_triangle.txt:3]: Is it meant to be `-filled`?

File: src/assets/usage-examples-code/graphics/fill_triangle/fill_triangle.cpp [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    open_window("Fill triangle Example", 800, 600);
+    clear_screen();
+    fill_triangle(COLOR_RED, 100, 100, 200, 200, 300, 100);
+    refresh_screen();
+
+    delay(5000);
+
+    close_all_windows();
+
+    return 0;
```
Comment (hugglesfox) [inline src/assets/usage-examples-code/graphics/fill_triangle/fill_triangle.cpp:14]: The code spacing here is a bit weird. My recommendation would be to group things around the `fill_triangle` function to emphasize it. E.g.

```c++
    open_window("Fill triangle Example", 800, 600);
    clear_screen();

    fill_triangle(COLOR_RED, 100, 100, 200, 200, 300, 100);
    refresh_screen();

    delay(5000);
    close_all_windows();

    return 0;
```

**Review by hugglesfox — APPROVED** [review]
A couple of nit picks in the review comments but looks good!

**Comment by sharvani28** [comment]
Made changes as per the suggesstion given by Hayley.

**Review by hugglesfox — APPROVED** [review]
Thanks. Looks good!

**Review by sananoureen35 — APPROVED** [review]

**Comment by omckeon** [comment]
Note for those reviewing usage examples - You need to test this in the browser to show it working there, not just the code separately. (@hugglesfox @s223126445 @sananoureen35 )

**Comment by sharvani28** [comment]
Updated the file names and included the screenshot in PR description.

**Pull request merged**